benchmarks/vectorization_strategy_benchmark.py

```diff
@@ -0,0 +1,700 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+# Setup/utils
+"""
+import time
+
+import matplotlib.pyplot as plt
```

Comment:
Is the benchmark a test or an example?

https://github.com/keras-team/keras-cv/blob/master/setup.py#L30